search/query_params.cpp

```diff
@@ -129,6 +133,7 @@ void QueryParams::AddSynonyms()
   {
     string const ss = ToUtf8(MakeLowerCase(token.GetOriginal()));
     auto const it = kSynonyms.find(ss);
+    LOG(LDEBUG, ("Searching for synonyms for ", ss));
```

Comment:
Is this log really necessary? It would make sense to log matched synonym.

Reply:
Nope. I'd meant to take that out when I finished debugging why S/South type matches weren't working. I'll push a new round of changes later today.

Reply:
Fixed in 1da320 - I just removed the logging statement.